<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParseHeaderClass</class>
 <widget class="QWidget" name="ParseHeaderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>726</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ParseHeaderWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="1,2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_List">
     <property name="title">
      <string>数据类型列表</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Select">
          <property name="text">
           <string>搜索</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Expand">
          <property name="text">
           <string>展开</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Import">
          <property name="text">
           <string>导入</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_Save">
          <property name="text">
           <string>保存</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox_Member">
     <property name="title">
      <string>数据类型成员</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
